frontend/src/pages/modelServing/screens/projects/utils.ts

```diff
@@ -667,3 +671,30 @@ export const getCreateInferenceServiceLabels = (
   }
   return undefined;
 };
+
+export const getPVC = async (
+  namespace: string,
+  servingRuntimeEditInfo?: ServingRuntimeKind,
+): Promise<PersistentVolumeClaimKind | undefined> => {
+  if (!servingRuntimeEditInfo) {
+    return undefined;
+  }
+
+  try {
+    const pvcName = servingRuntimeEditInfo.spec.volumes?.find(
+      (vol) => vol.persistentVolumeClaim?.claimName,
+    )?.persistentVolumeClaim?.claimName;
+
+    if (!pvcName) {
+      return undefined;
+    }
+
+    const pvcs = await getDashboardPvcs(namespace);
```

Comment:
Curious -- why not just fetch it if you have the K8s name? Why get DashboardPVCs and then filter?

Reply:
Good idea, I'll update it